flake.nix

```diff
@@ -216,13 +216,17 @@
           default = all-gcc;
         };
         apps = {
+          assigner = {
+            type = "app";
+            program = "${self.packages.${system}.evm-assigner}/bin/assigner/assigner";
+          };
           single-threaded = {
             type = "app";
             program = "${self.packages.${system}.proof-producer}/bin/proof-producer-single-threaded";
           };
           default = {
```

Comment:
default option is required?

Reply:
yes, a proof-producer script required it. I can change the script and remove the default, if you think, it's better

Reply:
Yes, thanks!. it's a little bit strange when required build some  application i any case

Reply:
fixed